data/mock_data.rb

```diff
@@ -14,11 +17,62 @@ def self.get_a_random_price
     (rand(1..120) + (rand(100) / 100.0))
   end
 
+  def self.get_a_random_status
+    case rand(3)
+      when 0
+        return 'pending'
+      when 1
+        return 'shipped'
+      when 2
+        return 'returned'
+    end
+  end
+
+  def self.invoices_as_mocks(invoice_hashes)
+    mocked_invoices = []
+    invoice_hashes.each do |invoice_hash|
+      raise NO_SELF_ERROR_MESSAGE if not block_given?
+      eg = yield
+      invoice_mock = eg.instance_double('Invoice',
+        id: invoice_hash[:id],
+        customer_id: invoice_hash[:customer_id],
+        merchant_id: invoice_hash[:merchant_id],
+        created_at: invoice_hash[:created_at],
+        updated_at: invoice_hash[:updated_at]
+      )
```

Comment:
This is a note for later, but we may need to consider how we're mocking times at some point. I've got it working well on items for now, though, so may not be necessary.